pkg/runtime/interfaces/cluster_configuration.go

```diff
@@ -3,15 +3,18 @@ package interfaces
 import (
 	"io/ioutil"
 
+	"github.com/flyteorg/flyteadmin/pkg/config"
+
 	"github.com/pkg/errors"
 )
 
 // Holds details about a cluster used for workflow execution.
 type ClusterConfig struct {
-	Name     string `json:"name"`
-	Endpoint string `json:"endpoint"`
-	Auth     Auth   `json:"auth"`
-	Enabled  bool   `json:"enabled"`
+	Name             string                   `json:"name"`
+	Endpoint         string                   `json:"endpoint"`
+	Auth             Auth                     `json:"auth"`
+	Enabled          bool                     `json:"enabled"`
+	KubeClientConfig *config.KubeClientConfig `json:"kube-client-config,omitempty"`
```

Comment:
I dont think we use this fomat (kebab-case?) but camelCase for all the configs in flyteadmin.
We can keep it consistent with other configs in this repo. Propeller and plugins repo i see use this format but may be we can keep using camelCase here

Reply:
done